http/server.ts

```diff
@@ -253,6 +254,8 @@ export class Server {
 
     this.#closed = true;
 
+    this.#acceptBackoffDelayAbortController.abort();
```

Comment:
Does it work to do this after the `for...of` loop a couple lines down?

Reply:
I made the change here: https://github.com/denoland/deno_std/pull/2732/commits/d7498295896b29ce0824d68f9c48010cbc6674cb